pootle/apps/pootle_store/fields.py

```diff
@@ -187,6 +187,7 @@ def _update_store_cache(self):
                 from pootle_store.filetypes import factory_classes
 
                 store_obj = factory.getobject(self.path,
+                                              localfiletype=self.instance.translation_project.project.localfiletype,
```

Comment:
`factory.getobject` doesn't have a `localfiletype` argument (see https://github.com/translate/translate/blob/master/translate/storage/factory.py#L165 ) so this commit doesn't apply. What do you tried to achieve here?